vagrant/provisioning/roles/start-arkcase/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+- name: start config server
+  become: yes
+  systemd:
+    name: "config-server"
+    state: started
+
+- name: wait for config server
+  wait_for:
+    connect_timeout: 15
+    delay: 5
```

Comment:
maybe if you `git pull origin develop` it will resolve these changes that you already made in the earlier PR